plugins/by-name/blink/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "blink";
+  originalName = "blink.cmp";
+  package = "blink.cmp";
+
+  maintainers = [ lib.maintainers.balssh ];
+
+  description = ''
+    blink.cmp is a completion plugin with support for LSPs and external sources while updating on every keystroke with minimal overhead (0.5-4ms async). It achieves this by writing the fuzzy searching in SIMD to easily handle >20k items. It provides extensibility via hooks into the trigger, sources and rendering pipeline. Plenty of work has been put into making each stage of the pipeline as intelligent as possible, such as frecency and proximity bonus on fuzzy matching, and this work is on-going.
```

Comment:
Can you add line breaks to make this readable with linewrapping disabled?

Usually I try and wrap semantically at punctuation such as `.` or `,` but the most important thing is to try and keep lines shorter than around 100-120 chars.

Not a hard-and-fast line-limit, but a good rule or thumb.

Note that single line breaks won't show up in the rendered docs. If you want user-visible newlines you should use either paragraph breaks (a blank line or double-linebreak) or a single-break (by ending the previous line with a `\`).